Mathlib/Dynamics/Flow.lean

```diff
@@ -171,9 +171,6 @@ def reverse : Flow τ α where
   map_zero' _ := by dsimp; rw [neg_zero, map_zero_apply]
 #align flow.reverse Flow.reverse
 
--- Porting note: add @continuity to Flow.toFun so that these works:
--- Porting note: Homeomorphism.continuous_toFun  : Continuous toFun  := by continuity
--- Porting note: Homeomorphism.continuous_invFun : Continuous invFun := by continuity
 @[continuity]
```

Comment:
From #14040: if you add
```suggestion
@[continuity, fun_prop]
```
then `fun_prop` can prove `continuous_*` below.

Reply:
Thanks for the hint! When I tried this, I needed to call `dsimp only` also. I've pushed this so you can take a look yourself.